pkgs/applications/office/todofi.sh/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "todofi.sh";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "hugokernel";
+    repo = "todofi.sh";
+    rev = "v${version}";
+    sha256 = "1gmy5inlghycsxiwnyyjyv81jn2fmfk3s9x78kcgyf7khzb5kwvj";
+  };
+
+  installPhase = ''
+    install -Dm 755 todofi.sh -t $out/bin
```

Comment:
Please wrap rofi and todo-txt so that they do not need to be installed also and the program just works.

Reply:
done in 'fixupPhase' could you take a look?